Tests/StackClusterTests.swift

```diff
@@ -171,10 +173,236 @@ class StackClusterTests: QuickSpec {
+                ]
+                
+                let vc = stack(with: nest)
+                vc?.loadViewIfNeeded()
+                vc?.view.layoutIfNeeded()
+
+                let childStack = vc?.childViewControllers[1] as? StackViewController
+                let targetVC = childStack?.childViewControllers[2]
+                
+                it("a target should be able to be centered") {
+                    childStack?.scrollToViewController(target: targetVC!, position: .center, animated: false)
+                    expect(vc).to(haveValidSnapshot())
+                }
+                
+                it("a target should be able to get left aligned") {
+                    childStack?.scrollToViewController(target: targetVC!, position: .beginning, animated: false)
+                    expect(vc).to(haveValidSnapshot())
+                }
+                
+                it("a target should be able to get right aligned") {
+                    childStack?.scrollToViewController(target: targetVC!, position: .end, animated: false)
+                    expect(vc).to(haveValidSnapshot())
+                }
+            }
+            
+            context("when scrolling vertically in a horizontal nested stack") {
+                let meta = ClusterLayout.StackConfig(axis: .horizontal)
+                let verticalChildMeta = ClusterLayout.StackConfig(axis: .vertical)
+                let size = CGSize(width: 40, height: 250)
+                let fixedSizeChildren = (1...3).map {
+                    labelComponent(title: String($0), color: .brown, labelSize: size)
+                }
+                let sizeNested = CGSize(width: 50, height: 150)
+                let children = [
+                    labelComponent(title: "1", color: .purple, labelSize: sizeNested),
+                    labelComponent(title: "2", color: .red, labelSize: sizeNested),
+                    labelComponent(title: "3", color: .yellow, labelSize: sizeNested),
+                    labelComponent(title: "4", color: .green, labelSize: sizeNested),
+                    labelComponent(title: "5", color: .blue, labelSize: sizeNested)
+                ]
+                
+                let nest = [
+                    ClusterLayout.stack(children: fixedSizeChildren, meta: meta),
+                    ClusterLayout.stack(children: children, meta: verticalChildMeta),
+                    ClusterLayout.stack(children: fixedSizeChildren, meta: meta)
+                ]
+                
+                let vc = stack(with: nest, meta: meta)
+                vc?.loadViewIfNeeded()
+                vc?.view.layoutIfNeeded()
+                
+                let childStack = vc?.childViewControllers[1] as? StackViewController
+                let targetVC = childStack?.childViewControllers[2]
+                
+                // TODO: WIP stack layout is not as expected
+                it("a target should be able to be centered") {
```

Comment:
@ChristianOrgler I looked into it and for me it seems like the StackViewController has a bug in this nested case or my test setup might need some adaptions to set the scroll views size correctly. It would cost me too much time to fix the StackViewController and is not in the scope of the `scroll to ViewController` feature.